pkg/obfuscator/domain_test.go

```diff
@@ -3,106 +3,61 @@ package obfuscator
 import (
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
 	"github.com/stretchr/testify/assert"
 )
 
 func TestDomainObfuscatorContents(t *testing.T) {
 	for _, tc := range []struct {
-		name   string
-		tlds   []string
-		input  []string
-		output []string
-		report map[string]string
+		name    string
+		domains []string
+		input   []string
+		output  []string
+		report  map[string]string
 	}{
 		{
-			name: "basic",
-			tlds: []string{".com", ".info", ".org"},
+			name:    "basic",
+			domains: []string{"redhat.com", "openshift.com", "okd.io"},
 			input: []string{
-				"received request on test.com",
-				"received request on https://test.com",
+				"received request on openshift.com",
+				"received request on https://docs.okd.io",
 			},
 			output: []string{
-				"received request on obfuscated0001.ext",
-				"received request on https://obfuscated0001.ext",
-			},
-			report: map[string]string{"test.com": "obfuscated0001.ext"},
-		},
-		{
-			name: "subdomain",
-			tlds: []string{".com", ".info", ".org"},
-			input: []string{
-				"received request on abc.test.com",
-				"received request on def.test.com",
-			},
-			output: []string{
-				"received request on abc.obfuscated0001.ext",
-				"received request on def.obfuscated0001.ext",
+				"received request on domain0000001",
```

Comment:
I'm actually wondering whether we can keep the TLD so it at least looks like a domain.

Reply:
The `soscleaner` doesn't. I don't see how it would be help anyway.

Reply:
with readability maybe? keep in mind that you also need to be on the receiving end of that must-gather at some point and that would make it look more naturally like a domain. In VSCode at least it gives me a domain/link highlighting.

Reply:
I believe the highlighting is because it contains a protocol prefix like `http://` or `https://` which will still be there.

Reply:
that's the underlining and making it clickable, but it also highlights things like quay.io properly without protocols. Anyway, why remove the ext if it was there before?

Reply:
talking about quay.io, shall we also keep an inclusion list of domains that are not private like openshift.io/com and quay?

Reply:
I removed it because it makes the logic more complex without substantial benefits. Because users might want to obfuscate domains which don't have extensions I don't think they would want it indicated if it has an extension or not.

Reply:
Since we have changed from a block-list to allow-list approach if the users don't want `openshift.io` they simply leave it out from the domains list.

Reply:
:+1: we can add it back if that's confusing for real users then.